lib/site_template/_config.yml

```diff
@@ -20,7 +20,8 @@ description: > # this means to ignore newlines until "baseurl:"
   line in _config.yml. It will appear in your document head meta (for
   Google search results) and in your feed.xml site description.
 baseurl: "" # the subpath of your site, e.g. /blog
-url: "http://example.com" # the base hostname & protocol for your site
+url: "" # the base hostname & protocol for your site, e.g. http://example.com
+        # [Required] for production environment.
```

Comment:
Added that based on [DirtyF's comment](https://github.com/jekyll/minima/issues/34#issuecomment-245447733)